cider/util/CiderBitUtils.h

```diff
@@ -183,6 +183,27 @@ inline size_t countSetBits(const uint8_t* bit_vector, size_t end) {
 
   return ans;
 }
+
+inline bool CheckBitVectorEq(const uint8_t* vec1, const uint8_t* vec2, int end) {
+  size_t i = 0, bytes = 0;
+
+  // do byte-batched check with memcmp
+  for (; i + 64 <= end; i += 64, bytes++)
```

Comment:
why 64? should be 8? and `bytes = end/8` would be fine?
1 byte  = 8 bits

Reply:
My fault. Copied directly from `countSetBits()` without thinking. `countSetBits()` did a `uint64_t` cast and used 64. But should be 8 here. Fixed.